types/notifyjs/index.d.ts

```diff
@@ -1,118 +1,111 @@
-// Type definitions for notifyjs 1.2.8
+// Type definitions for notifyjs 3.0.0
 // Project: https://github.com/alexgibson/notify.js
 // Definitions by: soundTricker <https://github.com/soundTricker>
+//                 NateScarlet <https://github.com/NateScarlet>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare var Notify: {
-    new (title : string , options? : notifyjs.INotifyOption): notifyjs.INotify;
+export default class Notify {
```

Comment:
Are you sure this is a default export and not `export = Notify`? See https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
https://github.com/alexgibson/notify.js/blob/30583643edea073fbc980a6c91113a3476d4c3a5/src/notify.js#L179
Source code is using `export default`

Reply:
`default` can be a function or else if i read this doc right: https://www.typescriptlang.org/docs/handbook/modules.html
And i think i am exporting a typescript class which will complie to javascript function at runtime.
Source code is using `export default` and previous version is not importable,  what situation this default property may cause problem?

Reply:
The source code was probably put through a transpiler which generated a commonjs module. But if this is declared as a `default` export, TypeScript code must use a default import, which outputs as `require("notifyjs").default`, which is undefined.